google/cloud/testing_util/mock_fake_clock.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google LLC
+// Copyright 2021 Google LLC
```

Comment:
I don't know if this copyright year should change if we're basically only moving the file. I suspect it should remain the same.